### PR TITLE
Added a demo-build script.

### PR DESCRIPTION
**Pull request by wpwood**
Added a demo-build script. Fixed a typo on the demo page. Fixed a typo in the server script.

### CODE REVIEW

**Comment by wpwood** [comment]
@adampickeral @underbyte @willcumbie What say you?

File: scripts/setup [hunk 1]
```diff
@@ -2,4 +2,4 @@
 
 # script/bootstrap: Resolve all dependencies required to develop and test canon-react components
 
-sudo npm install
```
Comment (underbyte) [inline scripts/setup]: :+1: for not sudoing

**Comment by underbyte** [comment]
LGTM

**Comment by adampickeral** [comment]
scripts/server already builds the demo files and starts an http server serving them up

**Comment by wpwood** [comment]
@adampickeral It didn't when I ran it this morning. No demo/bundle.js was created.

**Comment by adampickeral** [comment]
@wpwood make a change and it will.

you can add this line `./node_modules/.bin/grunt demo-build` before this line `./node_modules/.bin/grunt watch & node node_modules/.bin/http-server` in `scripts/server` and it will build it when you first run the script

**Comment by adampickeral** [comment]
i tried checking this out and i can't install without the sudo command

**Comment by adampickeral** [comment]
i guess it works if i run `sudo scripts/setup`... but sudo is still required

**Comment by wpwood** [comment]
@adampickeral Dunno. You shouldn't have to sudo npm unless you're installing something globally. Usually the issue is that you ran npm with sudo sometime in the past and forced a node module to be owned by root.

**Comment by adampickeral** [comment]
ok i think i fixed my sudo issue. get rid of the demo-build script and update the server script and :+1:

**Comment by wpwood** [comment]
@adampickeral I did

```
./node_modules/.bin/grunt demo-build && ./node_modules/.bin/grunt watch & node node_modules/.bin/http-server
```

so that the server command will exit if the demo-build fails.

**Comment by wpwood** [comment]
I'd also love to rename the branch, but I think it will require a new pull request and losing the discussion :)

**Comment by adampickeral** [comment]
cool, no need to rename the branch. do squash those two commits into one, and follow the commit message guidelines here: https://github.com/rackerlabs/canon-react/blob/master/CONTRIBUTING.md (this would be a 'chore' type of change). you can look at some of the recent merges for example commit messages

**Comment by adampickeral** [comment]
LGTM

**Pull request merged**